documentation/writing_plugins.md

```diff
@@ -34,27 +35,31 @@ A plugin can implement hooks as tasks. Bolt will search for tasks in a plugin mo
 
 In this case the plugin will implement two hooks: the `resolve_reference` hook that is explicitly defined and the `secret_decrypt` hook that is discovered from the task name. Bolt passes two metaparameters to all task hooks: `_config`, which contains the plugin configuration, and `_boltdir`, which contains the path to the current Boltdir.
 
-**`resolve_reference` tasks**
+### `resolve_reference` tasks
```

Comment:
`###` will cause these to be included in the table of contents. Is that desired? It definitely renders quite poorly without this change at least...

Reply:
Alternatively we can use `####`, but I think that's equivalent to just bolding... I'm not sure there's any harm in having this in the ToC, though, especially since it's fairly small for this page.

Reply:
Just a heads up - `####` would cause the docs build to break unless it was preceded with a `###` - Just a weird DITA-OT idiosyncrasy.

Reply:
Oh, good to know! 👍